src/Extensions/Compatability/Logger/Category.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+
+namespace Microsoft.Omex.Extensions.Compatability.Logger
```

Comment:
Typo. Replace every occurence, including this file's name.

Reply:
Thank you for noticing, all renamed